<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>记账软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabLedger">
       <attribute name="title">
        <string>记账界面</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>新增记账记录</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>记账日期：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDateEdit" name="dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
               <property name="date">
                <date>
                 <year>2023</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>当月工资：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" rowspan="2">
              <widget class="QDoubleSpinBox" name="salarySpinBox">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="minimum">
                <double>0.000000000000000</double>
               </property>
               <property name="maximum">
                <double>9999999.990000000223517</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0" rowspan="2">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>当月开支：</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1" rowspan="2">
              <widget class="QDoubleSpinBox" name="expenseSpinBox">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="minimum">
                <double>0.000000000000000</double>
               </property>
               <property name="maximum">
                <double>9999999.990000000223517</double>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>备注：</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="noteLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>当前总存款金额：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="totalDepositSpinBox">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="minimum">
                <double>0.000000000000000</double>
               </property>
               <property name="maximum">
                <double>9999999.990000000223517</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>定期余额：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" rowspan="2">
              <widget class="QDoubleSpinBox" name="fixedDepositSpinBox">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="minimum">
                <double>0.000000000000000</double>
               </property>
               <property name="maximum">
                <double>9999999.990000000223517</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0" rowspan="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>当月存款：</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1" rowspan="2">
              <widget class="QDoubleSpinBox" name="monthlyDepositSpinBox">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="minimum">
                <double>0.000000000000000</double>
               </property>
               <property name="maximum">
                <double>9999999.990000000223517</double>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>当月可支配额度：</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QDoubleSpinBox" name="disposableAmountSpinBox">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="minimum">
                <double>0.000000000000000</double>
               </property>
               <property name="maximum">
                <double>9999999.990000000223517</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="saveButton">
             <property name="text">
              <string>保存记录</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabChart">
       <attribute name="title">
        <string>记账曲线图界面</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QChartView" name="chartView" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
